rfcs/0051-close-stale-issues.md

```diff
@@ -0,0 +1,74 @@
+---
+feature: close-stale-issues
+start-date: 2019-08-24
+author: Ryan Mulligan
+co-authors: (find a buddy later to help our with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Close stale Nixpkgs issues and pull requests (hereafter both referred
+to as simply "issues") on GitHub using an application provided by
+GitHub.
+
+# Motivation
+[motivation]: #motivation
+
+The Nixpkgs GitHub page has a large number of open issues causing
+community angst and misrepresenting the responsiveness of the project.
+
+By closing stale issues, we can (in an automated way) refocus our
+efforts on the ones that have at least one person interested in them.
+
+# Detailed design
+[design]: #detailed-design
+
+1. Use the [Stale](https://github.com/marketplace/stale) application
+   provided by GitHub on the [Nixpkgs
+   repository](https://github.com/NixOS/nixpkgs).
+2. Start by using the following `.github/stale.yml` configuration
+   file:
+
+   ```
+   # Number of days of inactivity before an issue becomes stale
+   daysUntilStale: 60
```

Comment:
I think it must be not less than 6 months. One year is even better.

In this way, it'll help get rid of issues that can't be reproduced anymore, but also it'll not cause frustration of issue creator.